tasks/tasks.go

```diff
@@ -32,6 +32,36 @@ import (
 	"github.com/ystia/yorc/log"
 )
 
+type anotherLivingTaskAlreadyExistsError struct {
+	taskID   string
+	targetID string
+	status   string
+}
+
+func (e anotherLivingTaskAlreadyExistsError) Error() string {
+	return fmt.Sprintf("Task with id %q and status %q already exists for target %q", e.taskID, e.status, e.targetID)
+}
+
+// NewAnotherLivingTaskAlreadyExistsError allows to create a new anotherLivingTaskAlreadyExistsError error
+func NewAnotherLivingTaskAlreadyExistsError(taskID, targetID, status string) error {
```

Comment:
Sounds weird to do this. This means that `anotherLivingTaskAlreadyExistsError` should live in the collector package. But this will break the compatibility promise.